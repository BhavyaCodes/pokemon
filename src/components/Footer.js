import React from "react";
import footerStyle from "../assets/css/footerStyle.css"
import logo from "../assets/pokemon_logo.png";
import githubLogo from "../assets/github-logo.png";


function Footer() {
    return (
        <div className="footer">
            <div className="footer-logo">
                <img src={logo} alt="logo" width="100px" height="40px" />
            </div>

            <div className="logo-container">
                <div className="github-logo">
                    <a href="https://github.com/Juggernaut9/pokemon">
                        <img src={githubLogo} alt="logo" width="35px" height="35px" />
                    </a>
                </div>
            </div>
        </div>
    );

}

export default Footer;
