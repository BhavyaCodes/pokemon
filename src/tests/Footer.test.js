import React from "react";
import { render } from "@testing-library/react";

import Footer from "../components/Footer";

it("renders Footer component without crashing", () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
