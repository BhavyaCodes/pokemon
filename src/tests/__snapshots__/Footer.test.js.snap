// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component without crashing 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footer-logo"
    >
      <img
        alt="logo"
        height="40px"
        src="pokemon_logo.png"
        width="100px"
      />
    </div>
    <div
      class="logo-container"
    >
      <div
        class="github-logo"
      >
        <a
          href="https://github.com/Juggernaut9/pokemon"
        >
          <img
            alt="logo"
            height="35px"
            src="github-logo.png"
            width="35px"
          />
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
